plugin/al/translators/pxrUsdTranslators/tests/testPxrTranslators.py

```diff
@@ -21,7 +21,8 @@ def testExportProxyShapes(self):
         import AL.usdmaya
         
         tempFile = tempfile.NamedTemporaryFile(
-            suffix=".usda", prefix="AL_USDMayaTests_exportProxyShape_", delete=True)
```

Comment:
NamedTemporaryFile will not work on Windows the way it was being used in these tests.  Keeping the file open will stop the command from being able to write to it later on, so my fix is to use the function to create the file but then immediately close it.  At the end of the tests we can then remove it.